src/Core/System/Snippet/SnippetValidatorInterface.php

```diff
@@ -3,9 +3,18 @@
 namespace Shopware\Core\System\Snippet;
 
 use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\System\Snippet\Struct\SnippetValidationStruct;
 
 #[Package('discovery')]
 interface SnippetValidatorInterface
 {
+    /**
+     * @return array<string, array<string, mixed>>
+     */
     public function validate(): array;
+
+    /**
+     * @deprecated tag:v6.8.0 - Will be removed, use validate() instead
+     */
+    public function getValidation(): SnippetValidationStruct;
```

Comment:
you cannot simply add methods to an interface :see_no_evil: this will break every implementation of this interface outside of the core

Reply:
Thought it'd be okay because it's in a DI container, but kay.